release/pkg/manager/calico/manager.go

```diff
@@ -380,6 +381,11 @@ func (r *CalicoManager) PreReleaseValidate(ver string) error {
 		return fmt.Errorf("Attempt to release from a shallow clone is not possible")
 	}
 
+	// Check that code generation is up-to-date.
+	if _, err := r.runner.RunInDir(r.repoRoot, "make", []string{"generate", "get-operator-crds", "check-dirty"}, nil); err != nil {
+		return fmt.Errorf("code generation error: %s", err)
```

Comment:
can you include the solution for this in the error mesage.

Reply:
@radTuti done